stdlib/tests/collections/mmr.rs

```diff
@@ -1,3 +1,4 @@
+use processor::Host;
```

Comment:
Do we need this import here because of the `build_test!` macro? If so, is there a way to get the `Host` from `test_utils`?

Reply:
It was for the `<TestHost as Host>::advice_provider()`; I added it as a method directly to `TestHost` instead.